modules/nf-core/bbmap/filterbyname/main.nf

```diff
@@ -0,0 +1,72 @@
+process BBMAP_FILTERBYNAME {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bbmap:39.01--h5c4e2a8_0':
+        'biocontainers/bbmap:39.01--h5c4e2a8_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val(names_to_filter)
+    val(output_format)
+    val(interleaved_output)
+
+    output:
+    tuple val(meta), path("*.${output_format}"), emit: reads
+    tuple val(meta), path('*.log')             , emit: log
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def input  = meta.single_end ? "in=${reads}" : "in=${reads[0]} in2=${reads[1]}"
+    def output = (meta.single_end || interleaved_output) ?
+        "out=${prefix}.${output_format}" :
+        "out1=${prefix}_1.${output_format} out2=${prefix}_2.${output_format}"
+    def names_command = names_to_filter ? "names=${names_to_filter}": ""
+
+    def avail_mem = 3
+    if (!task.memory) {
+        log.info '[filterbyname] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
+    } else {
+        avail_mem = task.memory.giga
+    }
+
+    """
+    filterbyname.sh \\
+        -Xmx${avail_mem}g \\
+        $input \\
+        $output \\
+        $names_command \\
+        $args \\
+        &> ${prefix}.log
```

Comment:
Can we use `tee` for this please so the nextflow log/stdout can also see any logged error messages. https://nf-co.re/docs/guidelines/components/modules#capturing-stdout-and-stderr

Reply:
Hopefully the tee I've just tried to write works.